georocket-server/src/main/java/io/georocket/index/elasticsearch/ElasticsearchClient.java

```diff
@@ -111,12 +111,27 @@ public void close() {
    */
   public Observable<JsonObject> beginScroll(JsonObject query, int pageSize,
       String timeout) {
+    return beginScroll(query, new JsonObject(), pageSize, timeout);
+  }
+
+  /**
+   * Perform a search and start scrolling over the result documents
+   * @param query the query to send
+   * @param aggregation the aggregation function to apply
```

Comment:
I'm not sure if 'aggregation function' is a correct (Elasticsearch) term. I think they're just called 'aggregations'. Please use the correct term here.